pkg/tfvars/azure/azure.go

```diff
@@ -128,6 +130,10 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 		VMArchitecture:                  vmarch,
 	}
 
+	if sources.ImageRelease != "" {
+		cfg.ImageRelease = sources.ImageRelease
+	}
```

Comment:
Hm... do we need this or can you just add this to the literal above? 

Will sources.ImageRelease ever be ""? Even if it is I think the logic would be the same, because the empty value of cfg.ImageRelease is ""

Reply:
yes removed the release image checks.